modules/planning/math/curve1d/piecewise_quintic_spiral_path.h

```diff
@@ -0,0 +1,60 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file
+ **/
+
+#ifndef MODULES_PLANNING_MATH_CURVE1D_PIECEWISE_QUINTIC_SPIRAL_PATH_H_
+#define MODULES_PLANNING_MATH_CURVE1D_PIECEWISE_QUINTIC_SPIRAL_PATH_H_
+
+#include <vector>
+
+#include "modules/planning/math/curve1d/curve1d.h"
+#include "modules/planning/math/curve1d/quintic_spiral_path.h"
+
+namespace apollo {
+namespace planning {
+
+class PiecewiseQuinticSpiralPath : public Curve1d {
+ public:
+  PiecewiseQuinticSpiralPath();
+
+  virtual ~PiecewiseQuinticSpiralPath() = default;
+
+  void Append(QuinticSpiralPath spiral_path);
+
+  double Evaluate(const std::uint32_t order,
+      const double param) const override;
+
+  double DeriveKappaS(const double s) const;
+
+  double ParamLength() const override;
+
+  std::string ToString() const override { return ""; };
```

Comment:
you can return the class name here, instead of nothing.

Reply:
OK